nginx/common.sls

```diff
@@ -22,6 +22,11 @@
 /usr/share/nginx:
   file:
     - directory
+    
+apache2:
```

Comment:
Formula does not support running nginx + apache2 + salt-nginx-formula concurrently and Tobio says *we should not break this* .. I fully agree.  This is a bug.  Formula should ensure nginx installs without breaking UC required by users of the formula. According to Saltstack documentation, service.dead(apache2) enforces a transient state - if there is a problem with the patch thats one thing - but saying this plausible and widely reported UC is not supported scenario by formula is wrong. This is a formula bug.